server/src/test/java/org/spine3/server/BoundedContextShould.java

```diff
@@ -261,8 +258,8 @@ public TaskAdded handle(AddTask cmd, CommandContext ctx) {
         private void event(ProjectCreated event) {
 
             final Project newState = Project.newBuilder(getState())
-                    .setProjectId(event.getProjectId())
-                    .setStatus(STATUS_NEW)
```

Comment:
@AlexanderLitus, it's what I told you about yesterday. We wouldn't have status as a string in the production code. So shouldn't we do such a thing in tests.